beacon-chain/db/filesystem/blob.go

```diff
@@ -148,6 +153,7 @@ func (bs *BlobStorage) Get(root [32]byte, idx uint64) (blocks.VerifiedROBlob, er
 	if err != nil {
 		return blocks.VerifiedROBlob{}, err
 	}
+	blobFetchLatency.Observe(time.Since(startTime).Seconds())
```

Comment:
```suggestion
	blobFetchLatency.Observe(time.Since(startTime).Milliseconds())
```

Also, this might be better to do in a `defer` here or change this to Observe after the value is returned from BlobSidecarNoop